parquet-hadoop/src/test/java/org/apache/parquet/hadoop/vector/TestParquetVectorReader.java

```diff
@@ -566,7 +597,7 @@ public void testReadAll() throws Exception {
 
     } finally {
       if (reader != null)
-        reader.close();
+        ((Closeable) reader).close();
```

Comment:
Like other readers in Parquet (``ParquetReader`` and ``ParquetRecordReader``) what do you think about having a ``close()`` method in the ``VectorizedReader`` interface (by extending `java.io.Closeable`)? With that the caller won't have to cast the reader to ``Closeable`.